docs/topics/proc_running-the-containerless-mta-cli.adoc

```diff
@@ -34,14 +34,14 @@ NOTE: If you do not set `JVM_MAX_MEM`, the analysis might hang.
 +
 [source,terminal,subs="attributes+"]
 ----
-$ mv <mta_cli_zip>/<os>-kantra /usr/bin
+$ mv <mta_cli_zip>/<os>-mta-cli /usr/bin
```

Comment:
this is correct, only concern is that normally user will require to use `sudo` to copy to `/usr/bin`. Alternative is putting mta-cli into `$HOME/bin` and making sure it is a part of `$path` system variable

Reply:
Maybe it can mention moving the binary to your $PATH, and leave out the `/usr/bin` example. Or make it clear that that command is just an example.